Tests/UnitTests/Plugins/Json/MaterialMapJsonConverterTests.cpp

```diff
@@ -8,7 +8,7 @@
 
```

Comment:
The test is even called `MaterialMapJsonConverterTests` - so the class and file should be called `MaterialMapJsonConverter`.